### ISSUE TITLE
network data not working

### ISSUE THREAD

**Issue by omerlh** [kubecost/cost-model#257]
I deployed cost model, and the network pods. I can see network metrics on Prometheus (for example, kubecost_pod_network_egress_bytes_total) - but it still not working on the portal. When trying to get /model/networkTraffic?namespace=<>, an error is returned:
{"code":500,"data":null,"message":"Failed to locate network pods"}

I'm not sure where is error is thrown from, and I couldn't find this log anywhere...

**Comment by dwbrown2** [comment]
@omerlh thanks for reporting. Can you check and report the status of these tests?

1. Confirm the `kubecost-network-costs` pods are Running. If these pods are not in a Running state, kubectl describe them and/or view their logs for errors.
2. Ensure kubecost-networking target is Up in your Prometheus Targets list. View any visible errors if this target is not Up.
3. Verify Network Costs are available in your Kubecost Allocation view. View your browser's Developer Console on this page for any access/permissions errors if costs are not shown.

Also, what version of Kubecost are you running?

**Comment by omerlh** [comment]
Yes to all questions :) 

Kubecost version:

* gcr.io/kubecost1/kubecost-network-costs:v8
* ajaytripathy/kubecost-cost-model:prod-1.49.3
* ajaytripathy/kubecost:prod-1.49.3
* ajaytripathy/kubecost-frontend:prod-1.49.3

I used the manifest file deployment option, so I might have outdated images

**Comment by omerlh** [comment]
Yes to all questions :) 

Kubecost version:

* gcr.io/kubecost1/kubecost-network-costs:v8
* ajaytripathy/kubecost-cost-model:prod-1.49.3
* ajaytripathy/kubecost:prod-1.49.3
* ajaytripathy/kubecost-frontend:prod-1.49.3

I used the manifest file deployment option, so I might have outdated images

**Comment by dwbrown2** [comment]
Hi @omerlh, I'm still not clear what behavior you are experiencing... could you share a screenshot of `/network.html?ns=kube-system&window=1d`? And/or the Allocation view. You should have data here if metrics are available in Prometheus...

**Comment by dwbrown2** [comment]
Two other items that might be helpful:

1. can you share how you installed our product + enabled network costs? 
2. what do you get with this command? `kubectl get pods -l app=kubecost-network-costs --all-namespaces`

**Comment by omerlh** [comment]
This is the issue I'm having:
![image](https://user-images.githubusercontent.com/6730584/71616435-f278b900-2bbe-11ea-9ba7-b42b5f630f30.png)
When checking in devtools, I can see the following error:
```
Failed to load resource: the server responded with a status of 500 (Internal Server Error)
```
Which is the first error I reported.

I wanted to install it with helm, but didn't want to sign the license. So I used the flat file manifest, changed it a bit and added to it the network cost daemonset. This is the pods I have right now:
![image](https://user-images.githubusercontent.com/6730584/71616568-92cedd80-2bbf-11ea-96a6-1636603e2679.png)

I don't see the pods you mentioned - maybe I'm missing something...
